packages/shared/src/utils/contract/contract.tsx

```diff
@@ -159,10 +159,12 @@ export const isVanillaFxContract = (contract_type = '', symbol = '') =>
     isVanillaContract(contract_type) && VANILLA_FX_SYMBOLS.includes(symbol as typeof VANILLA_FX_SYMBOLS[number]);
 
 export const isSmartTraderContract = (contract_type = '') =>
-    /RUN|EXPIRY|RANGE|UPORDOWN|ASIAN|RESET/i.test(contract_type);
+    /RUN|EXPIRY|RANGE|UPORDOWN|ASIAN|TICK/i.test(contract_type);
```

Comment:
Will Reset Call/Put work? You removed it from regexp

Reply:
Sorry, this was due to merge conflicts, I will put it back

Reply:
I fixed the problem @kate-deriv 🙏🏻